ddtrace/propagation/http.py

```diff
@@ -1147,7 +1137,14 @@ def my_controller(url, headers):
                     context = HTTPPropagator._resolve_contexts(contexts, styles_w_ctx, normalized_headers)
                     if config.propagation_http_baggage_enabled is True:
                         _attach_baggage_to_context(normalized_headers, context)
-                    return context
+
+            # baggage
+            if config._propagation_style_extract and _PROPAGATION_STYLE_BAGGAGE in config._propagation_style_extract:
+                baggage_context = _BaggageHeader._extract(normalized_headers)
+                if baggage_context:
+                    context._baggage = baggage_context._baggage
```

Comment:
If there are no valid headers for the other propagation styles specified I think we can end up in a situation where context is None, in which case this would throw an error. We should check if there's a context, if so add baggage, if not return the baggage context value I think?